docs/node/daemon/config/settings.mdx

```diff
@@ -144,39 +160,48 @@ Here is an example of a `config.toml` file with complete configuration:
   <summary>sample config.toml file </summary>
 
 ```yaml
-# This is a TOML config file.
-# For more information, see https://github.com/toml-lang/toml
 
-# NOTE: Any path below can be absolute (e.g. "/var/myapp/data") or
+# NOTE: Any path below can be absolute (e.g. "/var/myawesomeapp/data") or
 # relative to the home directory (e.g. "data")
 
 # Root Directory Structure:
 # RootDir/
 #   |- config.toml   (app and chain configuration for running the kwild node)
-#   |- genesis.json  (genesis file for the network)
 #   |- private_key   (node's private key)
 #   |- abci/
-#   |   |- addrbook.json  (peer routable addresses for the kwild node)
-#   |   |- data/          (blockchain files: blockstore.db, state.db, etc)
+#   |   |- config/
+#   |   |   |- genesis.json   (genesis file for the network)
+#   |   |   |- addrbook.json  (peer routable addresses for the kwild node)
```

Comment:
This change isn't quite correct.  The genesis.json that users care about in most cases is still in RootDir.  This abci/config folder contains active config in the cometbft format.  This README might help explain better:

```
$  cat ~/.kwild/abci/config/README 
This config folder is used to echo the in-memory CometBFT configuration
that is generated from kwild's config and genesis files. This folder and the files
within are useful for running certain debugging commands with the official cometbft
command line application (github.com/cometbft/cometbft/cmd/cometbft).

WARNING: These files are overwritten on kwild startup
```


Also, afaik, addrbook.json is still in abci not abci/config.

Reply:
Okay - FYI, this is copy and paste from the autogen config.toml with `kwild --autogen` on v0.8.1. We may want to update the comments in that doc.

cc @charithabandi @brennanjl